src/components/NavBar.js

```diff
@@ -2,7 +2,9 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import { connect } from 'react-redux';
 import LogOut from './auth-components/logout';
+import ConnectivityMessage from './ConnectivityMessage';
 import './component.css';
+
 export class NavBar extends React.Component {
   render() {
     let logOutButton;
```

Comment:
Also add button seems to be gone now or rather the plus image, not sure if from this branch or a previous this was merged from. 

Also delete list button not working, but may be from previous branch.

Reply:
The add button being gone is unrelated to this PR (check dev/master). Can you explain delete list not working more?

Reply:
Nevermind, I can reproduce. Added to my todo list